diff --git a/node_modules/bulma/sass/utilities/initial-variables.sass b/node_modules/bulma/sass/utilities/initial-variables.sass
index 0650ce3..207d358 100644
--- a/node_modules/bulma/sass/utilities/initial-variables.sass
+++ b/node_modules/bulma/sass/utilities/initial-variables.sass
@@ -52,7 +52,7 @@ $block-spacing: 1.5rem !default
 // The container horizontal gap, which acts as the offset for breakpoints
 $gap: 32px !default
 // 960, 1152, and 1344 have been chosen because they are divisible by both 12 and 16
-$tablet: 769px !default
+$tablet: 768px !default 
 // 960px container + 4rem
 $desktop: 960px + (2 * $gap) !default
 // 1152px container + 4rem
